### ISSUE TITLE
Searching for Chinese is not working correctly.

### ISSUE THREAD

**Issue by YiqiXu** [XmacsLabs/mogan#222]
In Mogan v.1.0.2, searching for Chinese is not working correctly.

**Comment by darcy-shen** [comment]
Click for the first time:
```
;#<lambda (what)>: not enough arguments: ()
; (apply r args)
; (key-press key)                            ; key: "F1"
; ((with after? (in-inactive-reference?) (w... ; before?: #f
```

**Comment by darcy-shen** [comment]
Click for the second time, it shows but it crashes when we put the Chinese in:
```
Throwing segmentation fault
-------------------------------------------------
Error message:
  segmentation fault

System information:
  TeXmacs version  : 2.1.1
  Built by         : sadhen
  Building date    : 2022-05-15T21:38:43
  Operating system : Debian GNU/Linux bookworm/sid
  Processor        : x86_64
  Crash date       : 2022年 05月 15日 星期日 21:45:06 HKT

Editor status:
  Root path          : [ 1 ]
  Current path       : [ 1, 1, 0, 0, 1, 1 ]
  Shifted path       : [ 1, 1, 0, 0, 2, 0 ]
  Physical selection : [ 1, 1, 0, 0, 1, 1 ] -- [ 1, 1, 0, 0, 1, 1 ]

Backtrace of C++ stack:
  Mogan : () + 0x290afa
  Mogan : () + 0x12da65
  Mogan : () + 0xfd9d3
  /lib/x86_64-linux-gnu/libc.so.6 : () + 0x38920
  Mogan : () + 0x96ae28
  Mogan : () + 0x44ef12
  /usr/lib/x86_64-linux-gnu/libQt5Widgets.so.5 : QWidget::event(QEvent*) + 0x20e
  Mogan : () + 0x44cf5e
  /usr/lib/x86_64-linux-gnu/libQt5Widgets.so.5 : QWidget::event(QEvent*) + 0x20e
  Mogan : () + 0x451103
  /usr/lib/x86_64-linux-gnu/libQt5Widgets.so.5 : QApplicationPrivate::notify_helper(QObject*, QEvent*) + 0x7f
  Mogan : () + 0x109dc2
  /usr/lib/x86_64-linux-gnu/libQt5Core.so.5 : QCoreApplication::notifyInternal2(QObject*, QEvent*) + 0x12a
  /usr/lib/x86_64-linux-gnu/qt5/plugins/platforminputcontexts/libfcitx5platforminputcontextplugin.so : () + 0xffaf
  /usr/lib/x86_64-linux-gnu/libQt5Core.so.5 : () + 0x2e6133
  /usr/lib/x86_64-linux-gnu/libFcitx5Qt5DBusAddons.so.1 : fcitx::FcitxQtInputContextProxy::commitString(QString const&) + 0x32
  /usr/lib/x86_64-linux-gnu/libQt5Core.so.5 : () + 0x2e6133
  /usr/lib/x86_64-linux-gnu/libFcitx5Qt5DBusAddons.so.1 : fcitx::FcitxQtInputContextProxyImpl::CommitString(QString const&) + 0x32
  /usr/lib/x86_64-linux-gnu/libFcitx5Qt5DBusAddons.so.1 : fcitx::FcitxQtInputContextProxyImpl::qt_metacall(QMetaObject::Call, int, void**) + 0x63
  /usr/lib/x86_64-linux-gnu/libQt5DBus.so.5 : () + 0x2394b
  /usr/lib/x86_64-linux-gnu/libQt5Core.so.5 : QObject::event(QEvent*) + 0x2ae
  /usr/lib/x86_64-linux-gnu/libQt5Widgets.so.5 : QApplicationPrivate::notify_helper(QObject*, QEvent*) + 0x7f
  Mogan : () + 0x109dc2
  /usr/lib/x86_64-linux-gnu/libQt5Core.so.5 : QCoreApplication::notifyInternal2(QObject*, QEvent*) + 0x12a
  /usr/lib/x86_64-linux-gnu/libQt5Core.so.5 : QCoreApplicationPrivate::sendPostedEvents(QObject*, int, QThreadData*) + 0x187
  /usr/lib/x86_64-linux-gnu/libQt5Core.so.5 : () + 0x307a83
  /usr/lib/x86_64-linux-gnu/libglib-2.0.so.0 : g_main_context_dispatch() + 0x26b
  /usr/lib/x86_64-linux-gnu/libglib-2.0.so.0 : () + 0x54238
  /usr/lib/x86_64-linux-gnu/libglib-2.0.so.0 : g_main_context_iteration() + 0x2f
  /usr/lib/x86_64-linux-gnu/libQt5Core.so.5 : QEventDispatcherGlib::processEvents(QFlags<QEventLoop::ProcessEventsFlag>) + 0x64
  /usr/lib/x86_64-linux-gnu/libQt5Core.so.5 : QEventLoop::exec(QFlags<QEventLoop::ProcessEventsFlag>) + 0x12b
  /usr/lib/x86_64-linux-gnu/libQt5Widgets.so.5 : QDialog::exec() + 0x1fa
  Mogan : () + 0x477f06
  Mogan : () + 0x476f6e
  Mogan : () + 0x475ade
  Mogan : () + 0x28d91e
  Mogan : () + 0x28d62b
  Mogan : () + 0x29d2c0
  Mogan : () + 0x1bc0d1
  Mogan : () + 0x1d2b6d
  Mogan : () + 0xacc1da
  Mogan : () + 0xaf82a6
  Mogan : () + 0xa37c46
  Mogan : () + 0x1517a7
  Mogan : () + 0x1517e1
  Mogan : () + 0x1dfb8c
  Mogan : () + 0x29db35
  Mogan : () + 0x7a1406
  Mogan : () + 0x7a2b0b
  Mogan : () + 0x1abde9
  Mogan : () + 0x1cff77
  Mogan : () + 0xae777a
  Mogan : () + 0xaf165f
  Mogan : () + 0xa37c46
  Mogan : () + 0x1517a7
  Mogan : () + 0x151887
  Mogan : () + 0x1ddeb5
  Mogan : () + 0x7a481a
  Mogan : () + 0x480db3
  Mogan : () + 0x481e8c
  Mogan : () + 0x448d17
  Mogan : () + 0x1060cb
  /usr/lib/x86_64-linux-gnu/libQt5Core.so.5 : () + 0x2e6168
  /usr/lib/x86_64-linux-gnu/libQt5Core.so.5 : QTimer::timeout(QTimer::QPrivateSignal) + 0x3a
  /usr/lib/x86_64-linux-gnu/libQt5Core.so.5 : QObject::event(QEvent*) + 0x1cf
  /usr/lib/x86_64-linux-gnu/libQt5Widgets.so.5 : QApplicationPrivate::notify_helper(QObject*, QEvent*) + 0x7f
  Mogan : () + 0x109dc2
  /usr/lib/x86_64-linux-gnu/libQt5Core.so.5 : QCoreApplication::notifyInternal2(QObject*, QEvent*) + 0x12a
  /usr/lib/x86_64-linux-gnu/libQt5Core.so.5 : QTimerInfoList::activateTimers() + 0x3db
  /usr/lib/x86_64-linux-gnu/libQt5Core.so.5 : () + 0x306d44
  /usr/lib/x86_64-linux-gnu/libglib-2.0.so.0 : g_main_context_dispatch() + 0x26b
  /usr/lib/x86_64-linux-gnu/libglib-2.0.so.0 : () + 0x54238
  /usr/lib/x86_64-linux-gnu/libglib-2.0.so.0 : g_main_context_iteration() + 0x2f
  /usr/lib/x86_64-linux-gnu/libQt5Core.so.5 : QEventDispatcherGlib::processEvents(QFlags<QEventLoop::ProcessEventsFlag>) + 0x64
  /usr/lib/x86_64-linux-gnu/libQt5Core.so.5 : QEventLoop::exec(QFlags<QEventLoop::ProcessEventsFlag>) + 0x12b
  /usr/lib/x86_64-linux-gnu/libQt5Core.so.5 : QCoreApplication::exec() + 0x90
  Mogan : () + 0x480ace
  Mogan : () + 0x480b2b
  Mogan : () + 0x10149c
  Mogan : () + 0x151503
  Mogan : () + 0x10393e
  /lib/x86_64-linux-gnu/libc.so.6 : __libc_start_main() + 0xcd
  Mogan : () + 0xfd8fa

-------------------------------------------------
terminate called after throwing an instance of 'string'

```

**Comment by darcy-shen** [comment]
closed by #254
